spec/namespaces/indices.yaml

```diff
@@ -3828,6 +3830,17 @@ components:
       schema:
         type: boolean
       style: form
+    indices.get_mapping::query.index:
+      in: query
+      name: index
+      description: |-
+        Comma-separated list of data streams, indices, and aliases used to limit the request.
+        Supports wildcards (`*`).
+        To target all data streams and indices, omit this parameter or use `*` or `_all`.
+      required: true
```

Comment:
I missed this. it should have been false

Reply:
Yikes